Python/Image_Sketch/README.md

```diff
@@ -1,10 +1,20 @@
-# Image to Sketch
-The original image is converted into black and white sketched image.
-
-### Results:
-
-#### Original Image
-<img src ="https://github.com/sharur7/Rotten-Scripts/blob/sharur7/Python/Image_Sketch/obama.jpg?raw=true" alt="Original_image" width="280" height="300">
-
-#### Sketched Image
-<img src ="https://github.com/sharur7/Rotten-Scripts/blob/sharur7/Python/Image_Sketch/sketch.jpg?raw=true" alt="Original_image" width="280" height="300">
+# CONVERTING IMAGE TO PENCIL SKETCH
+
+### A RGB image can be converted into a pencil sketch using 4 easy steps!
+
+![img.jpg](img.jpg)

```

Comment:
Hey there. Upload all the images on an image hosting system, like imgur. And paste the links here

Reply:
Done. Please check now @vybhav72954.